Objects/moduleobject.c

```diff
@@ -866,6 +866,42 @@ _Py_module_getattro(PyModuleObject *m, PyObject *name)
     return _Py_module_getattro_impl(m, name, 0);
 }
 
+static int
+module_setattro(PyModuleObject *mod, PyObject *name, PyObject *value)
+{
+    PyObject *res;
```

Comment:
Can you please move the variable just where it's used? It's ok to have 2 variables (of the same name) in two different scopes.